### ISSUE TITLE
Unlimited Range Pointing

### ISSUE THREAD

**Issue by Drobe50** [ParadiseSS13/Paradise#13772]
**Issue Description**: Infinite Range Pointing

**What did you expect to happen**: Not being able to point through the entertainment console.

**What happened instead**: You can point from any range with the Librarians camera and a entertainment monitor.

**Why is this bad/What are the consequences**: You can point anywhere you aren't. 

**Steps to reproduce the problem**: Turn on librarians camera, go to entertainment monitor, point anywhere on screen.

**When did the problem start happening**: Unknown

**Extra information**: N/A

**Comment by stylemistake** [comment]
Was anyone able to see you pointing through cameras? If not, I don't see the issue.

**Comment by Drobe50** [comment]
> 
> 
> Was anyone able to see you pointing through cameras? If not, I don't see the issue.

Actually they can, it's not just a "Clientside" thing. I forgot to mention that.

**Comment by stylemistake** [comment]
I don't believe you, post video proofs from a bystanders perspective, preferably at very long range, and with obstruction between the camera user and the observer.

Why I think this isn't an issue - visibility rules still apply. If you can't see the person pointing at you, you don't see the pointing.

**Comment by stylemistake** [comment]
Ok, I retract my previous statement, I thought tgui camera consoles were merged here. :smiley:

**Comment by Contrabang** [comment]
I believe this has been fixed because I am not able to reproduce it with any of the current cameras, you're only able to point to areas in your current vision through cameras. However, I am not sure what PR fixed this possibly #19072? Leaving this open for a while to see if I can get any confirmations on this
